### DISCUSSION TITLE
Generated onnx has incorrect output

### DISCUSSION THREAD

**Discussion by Jubal-Cww** [RapidAI/RapidLatexOCR#9] [Q&A] [answered]
Thanks for your great work. And I must say that it has been incredibly valuable to me.

However, I have encountered an issue while following the steps mentioned in the [Model Conversion Notes](https://github.com/RapidAI/RapidLatexOCR/wiki/Model-Conversion-Notes#convert-the-decoderonnx). Even after inserting the export code into Latex-OCR and resolving the 'context' error, the resulting ONNX model is still producing incorrect output. When I tried using your provided ONNX model, it worked perfectly fine.

I would like to ask if you made any additional modifications to the conversion process that are not mentioned in the notes. I noticed the mention of Attention sentences in the Model Conversion Notes. Could you please provide guidance on which conditional statements should be commented out in order to ensure proper conversion?

I am grateful for any assistance you can provide in resolving this issue. Thank you for your time and help.

**Comment by SWHL** [comment]
In the next few days, I will start to reorganize the notes related to converting this model to make it easier for everyone to reproduce. 

Please wait a little longer!

**Comment by SWHL** [comment, marked as answer]
I have reorganized the conversion code into this project [ConvertLaTeXOCRToONNX](https://github.com/SWHL/ConvertLaTeXOCRToONNX).
Enjoy it!
